addons/rivet/devtools/dock/loading.gd

```diff
@@ -0,0 +1,9 @@
+extends VBoxContainer
+
+
+func _ready() -> void:
+	%CancelButton.pressed.connect(_on_cancel_button_pressed)
+
+func _on_cancel_button_pressed() -> void:
+	# TODO(forest): cancel cli command
```

Comment:
Note for myself, I'm curious if this would require sending a kill signal to the Godot thread running the task.